docs/ai-consumers/tokens.md

```diff
@@ -13,35 +13,22 @@ micro_nav: true
 ---
 
 ## Introduction to AGI Tokens
-The AGI Token is an ERC-20 token, which is hosted on the Ethereum Blockchain.
+
+* The AGI Token is an ERC-20 token, which is hosted on the Ethereum Blockchain.
+* AGI is the proprietary crypto currency token used by the SingularityNET platform
+* AGI can be stored in any ERC20-compatible wallet (for example [MyEtherWallet](https://www.myetherwallet.com/)) 
 
 SingularityNET uses the AGI Token for any paid calls to a service.
-To learn more about how to obtain Kovan or Ropsten Testnet AGI, click on the link [here](#agi-faucet).
 
-## Testnet AGI vs Mainnet AGI
-If you are unfamiliar with  Ethereum, it might be hard to understand between Testnet and Mainnet tokens. 
-However, to understand the different network tokens, you can follow principles listed below:
-* Testnet tokens are used to test software applications with, try out demos, or in our case the Beta.
-* Mainnet tokens are used for officially deployed software.
-* Testnet tokens never have a monetary value.
-* Mainnet tokens could have a monetary value.
-* Both tokens can be used to add extra utility to the software.
+## How can one get AGI Tokens to call a service ?
+   You could pay in dollars and get AGI tokens.
+   [Click here](/docs/ai-consumers/paypal)
 
-We only use Kovan and Ropsten Testnet AGI and the current beta stage. 
 
-To learn more about  Mainnet AGI Token (to be used in a later stage) refer to this link [here](https://etherscan.io/address/0x8eb24319393716668d768dcec29356ae9cffe285) with the address `0x8eb24319393716668d768dcec29356ae9cffe285`.
+To see transactions on AGI Token refer to this link [here](https://etherscan.io/address/0x8eb24319393716668d768dcec29356ae9cffe285) with the address `0x8eb24319393716668d768dcec29356ae9cffe285`.
 
 <div class="callout callout--warning">
-    <p><strong>NOTE</strong> Currently, AGI token is traded on some exchanges, and also the speculative secondary trading is considered against the ethics of AGI token and SingularityNET project. Therefore, it is not recommended to exchange trading in any manner.</p>
+    <p><strong>NOTE:  </strong> Currently, AGI token is traded on some exchanges, and also the speculative secondary trading is considered against the ethics of AGI token and SingularityNET project. Therefore, it is not recommended to exchange trading in any manner.</p>
 </div>
 
```

Comment:
Is it worth to mention the Faucet for getting the test tokens to play with ropsten network.